src/Api.js

```diff
@@ -1,40 +1,226 @@
-import cookie from 'react-cookies';
+// noinspection JSUnusedGlobalSymbols
 
-const backendUrl = 'http://localhost:7274';
+const backendUrl = 'http://localhost:8080';
```

Comment:
Please change this to the deployed heroku url before merging

Reply:
Thought this was a deployment URL. Will fix it